docs/reference/method/MongoDBModelCollectionInfo-getCappedMax.txt

```diff
@@ -28,6 +30,9 @@ Return Values
 The document limit for the capped collection. If the collection is not capped,
 ``null`` will be returned.
 
+This method is deprecated in favor of using
+:phpmethod:`MongoDB\\Model\\CollectionInfo::getOptions()`.
```

Comment:
May be more helpful to note that users should access the `max` property. I realize we can't show `getOptions()['max']` since the method is a link. Consider:

```
...getOptions()` and accessing the `max` key.
```